Makefile

```diff
@@ -1,72 +1,55 @@
-install_trunk:
-	$(eval trunk_installed=$(shell trunk --version > /dev/null 2>&1 ; echo $$? ))
-ifneq (${trunk_installed},0)
-	$(eval OS_NAME=$(shell uname -s | tr A-Z a-z))
-	curl https://get.trunk.io -fsSL | bash
-endif
+# <!-- please update this list in the Readme "Global dependencies" list if you add others
+GLOBAL_DEPENDENCIES = trunk poetry aws docker
 
-uninstall_trunk:
-	sudo rm -if `which trunk`
-	rm -ifr ${HOME}/.cache/trunk
+# --- Setup --- #
+bootstrap: bootstrap_check_dependencies bootstrap_configure_pyright
 
-create_env:
-	# Copy .env
-	cp .env.example .env
+bootstrap_check_dependencies:
+	@for dep in $(GLOBAL_DEPENDENCIES); do \
+		if ! command -v $${dep} &> /dev/null; then \
+			echo "$${dep} is not installed. See readme for links to instructions."; \
+			exit 1; \
+		fi \
+	done
 
-configure_pyright:
+bootstrap_configure_pyright:
 	trunk actions run configure-pyright
 
-setup_with_pyenv: install_trunk create_env ## Sets up a local dev environment using Pyenv
-	$(eval venv_name=$(shell  grep 'venv =' pyproject.toml | cut -d '"' -f 2 ))
-	if [ -n "$(venv_name)" ] && ! pyenv versions --bare | grep -q "^$(venv_name)$$"; then \
-		$(eval python_version=$(shell grep 'python =' pyproject.toml | cut -d '"' -f 2 | sed 's/^\^//')) \
-		$(eval pyenv_version=$(shell pyenv versions --bare | grep$(python_version) )) \
-		pyenv virtualenv $(pyenv_version) $(venv_name); \
-	fi
-	@eval "$$(pyenv init -)" && \
-	pyenv activate $(venv_name) && \
-	poetry install
 
-	make configure_pyright
-	
-check:
-	trunk fmt
-	trunk check
+# --- Dev ---#
+dev: dev_rds_dump
+	docker-compose -f docker-compose-dev.yml up
 
-build:
-	docker build -t navigator-admin-backend .
-
-build_dev:
-	docker compose build
-
-unit_test: build
-	docker compose run --rm navigator-admin-backend pytest -vvv tests/unit_tests
+dev_rds_dump:
+	[ ! -f ./dumps/navigator.sql ] && aws --profile staging s3 cp s3://cpr-staging-rds/dumps/navigator.sql ./dumps/ || echo 0
```

Comment:
This is currently a file auto generated every day manually. We will try to make this more automated in future.